### DISCUSSION TITLE
scaling based on internal http request

### DISCUSSION THREAD

**Discussion by havard024** [kedacore/http-add-on#930] [Q&A]
Hi,
does the project support scaling based on internal http requests?

eg external request -> service A -> service B (internal request between service A and service B does not trigger scale up)

**Comment by michel-numan** [comment]
I created a new one incase no one saw this. 
I need this too

**Comment by michel-numan** [comment]
@havard024 
You need to create a separate service for the app, and point the service to a ExternalName to your ingress controller. Then create an ingress object (or equivalent) which captures the name of the service, and routes to the keda-proxy service (Which is also an ExternalName service to the keda interceptor)
